src/arduino/blinkende_lysdiode/blinkende_lysdiode.md

```diff
@@ -188,33 +201,26 @@ lampen av og på med en knapp!
 
 ## Sjekkliste {.check}
 
-+ Finn en ledning og en knapp.
-+ Koble til bryteren som på bildet under:
++ Finn tre ledninger til, og en knapp.
```

Comment:
Skal være tre. Ikke nytt bildet, men erstatet det gamle som var der.

Koblet slik for å vise enklere hva som går til jord og ikke. Gamle bildet kunne bli forvirrende, da knappen og lyset brukte samme ground.